### PR TITLE
Github Action to create Docker image

### PR DESCRIPTION
**Pull request by tschumpr**
This action builds a docker image whenever a new release is published. The image is named after the release tag, e.g. tag _v.3.1.0-test_ results in the image _3.1.0.-test_.

### CODE REVIEW

**Comment by pedroetb** [comment]
Hi @tschumpr, thanks for your contribution!

I was trying to use this image <https://github.com/GeoWerkstatt/stac-browser/pkgs/container/stac-browser> (image `ghcr.io/geowerkstatt/stac-browser:3.1.0`) but it's not working. I'm getting only a white page and errors while loading `app.js` and `chunk-vendors.js`.

image

Same configuration is working for me when using an image built by myself from <https://github.com/radiantearth/stac-browser/tree/ae7e5cf1c98dc7cd9c03f5e1a4d333556e345b8e>.

**Comment by m-mohr** [comment]
What do my Docker magicians think about this? Sounds like a good idea, I just don't have a clue about it myself. @geospatial-jeff @waystilos @constantinius

**Comment by waystilos** [comment]
> What do my Docker magicians think about this? Sounds like a good idea, I just don't have a clue about it myself. @geospatial-jeff @waystilos @constantinius

lgtm! Happy new year!

**Review by waystilos — APPROVED** [review]
lgtm

**Comment by m-mohr** [comment]
Thanks, I'll merge it now.

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# *nix shell scripts always use LF (see .editorconfig)
```
Comment (m-mohr) [inline .gitattributes:1]: Why is this in here? I don't see an .sh file being added.

File: README.md [hunk 2]
```diff
@@ -241,6 +241,7 @@ STAC Browser supports some non-standardized extensions to the STAC specification
 2. A link with relation type `icon` and a Browser-supported media type in any STAC entity will show an icon in the header and the lists.
 
 ## Docker
+### Create custom image
```
Comment (m-mohr) [inline README.md:244]:
```suggestion

### Create a custom image
```

File: README.md [hunk 3]
```diff
@@ -257,6 +258,18 @@ To run the container:
 docker run -p 8080:8080 stac-browser:v1
 ```
 
+### Use existing image
```
Comment (m-mohr) [inline README.md:261]:
```suggestion
### Use an existing image
```

**Comment by m-mohr** [comment]
Looks good to me. Left 3 minor comments. Once these are resolved, we I can merge this @tschumpr. Thanks for the PR.

Do I need to enable Packages on GitHub or something like that?

File: .gitattributes (hunk 1 above)
Comment (m-mohr) [reply .gitattributes:1]: Generally makes sense to use LF for something that is intended for Linux systems, so let's keep it for now.

**Review by m-mohr — APPROVED** [review]

**Comment by m-mohr** [comment]
Thanks, let's see how it goes...

**Pull request merged**